src/main/java/org/folio/consortia/service/TenantManager.java

```diff
@@ -53,4 +53,19 @@ public interface TenantManager {
    */
   void delete(UUID consortiumId, String tenantId, TenantDeleteRequest tenantDeleteRequest);
 
+
+  /**
+   * Creates an identity provider for the member tenant in the execution context tenant realm.
+   *
+   * @param memberTenant the tenant to create the identity provider for
+   */
+  void createIdentityProvider(String memberTenant);
```

Comment:
In other methods of this interface we use id suffix, so lets use it here like memberTenantId to have the same conventions